### PR TITLE
filesize: support the ronna and quetta prefixes

### PR DESCRIPTION
**Pull request by bbolli**

### CODE REVIEW

**Comment by bbolli** [comment]
Hi! I have no idea how to add a "PR label".

**Comment by hugovk** [comment]
I'll add the PR label, thanks! (And it's an addition not skip.)

Please could you add some test cases so Y, TiB, and TB are still covered somehow?

**Comment by bbolli** [comment]
> Please could you add some test cases so Y, YiB, and YB are still covered somehow?

Done.

File: src/humanize/filesize.py [hunk 1]
```diff
@@ -65,15 +89,11 @@ def naturalsize(
...
-    if abs_bytes < base and not gnu:
-        return "%d Bytes" % bytes_
-
-    if abs_bytes < base and gnu:
-        return "%dB" % bytes_
-
-    for i, s in enumerate(suffix):
-        unit = base ** (i + 2)
+    if abs_bytes < base:
+        return ("%dB" if gnu else "%d Bytes") % bytes_
```
Comment (hugovk) [inline src/humanize/filesize.py:93]: Let's keep the previous two `if` statements.

Comment (bbolli) [reply src/humanize/filesize.py:93]: It seems weird to do the same `abs_bytes < base`compare twice...

File: src/humanize/filesize.py [hunk 2]
```diff
@@ -65,15 +89,11 @@ def naturalsize(
...
 
+    for i, s in enumerate(suffix, 2):
+        unit = base**i
```
Comment (ErickMwazonga) [inline src/humanize/filesize.py:96]: nit - Add space around `**`

Comment (hugovk) [reply src/humanize/filesize.py:96]: Let's not bother, Black will remove them.

**Comment by hugovk** [comment]
Thanks!

**Pull request merged**